### PR TITLE
[xibuild] Handle "incorrectly" cased msbuild property names 

### PR DESCRIPTION
**Pull request by radical**
Fixes https://github.com/mono/mono/issues/14765

### CODE REVIEW

File: tools/xibuild/Main.cs [hunk 1]
```diff
@@ -228,7 +228,8 @@ void SetToolsetProperty (string name, string value)
 				if (string.IsNullOrEmpty (value))
 					return;
 
-				var valueAttribute = toolsets.SelectSingleNode ($"property[@name='{name}']/@value");
+				// MSBuild property names are case insensitive
+				var valueAttribute = toolsets.SelectSingleNode ($"property[translate(@name, 'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 'abcdefghijklmnopqrstuvwxyz')='{name.ToLower()}']/@value");
```
Comment (spouliot) [inline tools/xibuild/Main.cs]: `ToLowerInvariant` ?

**Comment by radical** [comment]
```
MonoTouchFixtures.CoreMidi.MidiThruConnectionTests.ConnectionCreateTest: midi connection error
Expected: True
But was: False

MonoTouchFixtures.CoreMidi.MidiThruConnectionTests.FindTest: connections should not be null
Expected: not null
But was: null

MonoTouchFixtures.CoreMidi.MidiThruConnectionTests.GetSetParamsTest: midi connection error
Expected: True
But was: False
```

Are these expected/known? They don't look related, based on the errors. But please correct me if I'm wrong and I'll take a look.

**Comment by mandel-macaque** [comment]
@radical we had a similar issue, but was closed: https://github.com/xamarin/maccore/issues/658

**Comment by spouliot** [comment]
@mandel-macaque re-open the issue and link to it - then it can be merged (and likely backported)

**Comment by spouliot** [comment]
@monojenkins backport to d16-2

**Pull request merged**